client/src/main/java/io/pravega/client/stream/EventStreamWriter.java

```diff
@@ -59,14 +59,14 @@
      *         exponential backoff. So there is no need to attempt to retry in the event of an exception.
      */
     CompletableFuture<Void> writeEvent(String routingKey, Type event);
-
     /**
      * Start a new transaction on this stream. This allows events written to the transaction be written an committed atomically.
-     * Note that transactions can only be open for {@link EventWriterConfig#getTransactionTimeoutTime()}.
-     * 
+     * Note that transactions can only be open for {@link EventWriterConfig#transactionTimeoutTime}
```

Comment:
This looks like a private field. I doubt this is where we should be pointing the Javadoc link

Reply:
Similar comments for similar changes below.

Reply:
yeah. It was a compromise. I thought the information about the behavior based on the value was important, but the actual accessors were generated by lombok  dynamically at runtime. So a lot of warnings popped up  because the link to the accessor wasn't available statically during the javadoc creation. Talked with Tom about it and that was the best I could do for a @link. alternatively I suppose we can take out the @link annotation and have a flat reference to the accessor.